storybook/src/components/TextArea/TextArea.stories.tsx

```diff
@@ -0,0 +1,90 @@
+/**
+ * @license EUPL-1.2+
+ * Copyright Gemeente Amsterdam
+ */
+
+import { TextArea, TextAreaProps } from '@amsterdam/design-system-react'
+import { useArgs } from '@storybook/preview-api'
+import { Meta, StoryObj } from '@storybook/react'
+import { exampleParagraph } from '../shared/exampleContent'
+
+const paragraph = exampleParagraph()
+
+type StoryProps = TextAreaProps & { invalid?: boolean }
+
+const meta = {
+  title: 'Components/Forms/Text Area',
+  component: TextArea,
+  args: {
+    children: paragraph,
+    disabled: false,
+  },
+  argTypes: {
+    resize: {
+      control: {
+        type: 'select',
+      },
+      options: ['default', 'none', 'horizontal', 'vertical'],
```

Comment:
`'default'` causes a non-existant `amsterdam-text-area--resize-default` to be added

Reply:
Feels wrong to change a component to make options in storybook work but here you go.

Reply:
I agree, you kind of solved it the wrong way around 😅 You can give an option the value of undefined, which doesn't set the prop. 

[Similar to what we do here](https://github.com/Amsterdam/design-system/blob/develop/storybook/src/components/IconButton/IconButton.stories.tsx#L22-26)